recipes/voms/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "voms" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Use the release candidate as it's now very old (tagged Jan 2017)
+  # Main difference is support for newer OpenSSL versions and would need to be patched regardless
+  url: https://github.com/italiangrid/{{ name }}/archive/v{{ version }}-rc0.tar.gz
```

Comment:
conda supports pep440 and version strings with `rc` in them. You should probably add that to the version variable so people installing know that this is `rc` and not a releases stable version without having to look in the recipe.